src/utils/fsExists.ts

```diff
@@ -0,0 +1,10 @@
+import { promises as fs } from 'fs'
```

Comment:
`fs.exists` is deprecated with Node v12: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback